README.md

```diff
@@ -606,38 +650,36 @@ Follow the below steps to set up Prometheus and view metrics for Ballerina restf
    http://localhost:19090/
 ```
 
-NOTE:  Ballerina will by default have following metrics for HTTP server connector. You can enter following expression in Prometheus UI
+NOTE:  By default, Ballerina will have the following metrics for HTTP server connector. You can enter the following expressions in Prometheus UI:
```

Comment:
Minor correction:
"NOTE:  By default, Ballerina provides the following metrics for HTTP server connector. You can enter the following expressions in the Prometheus UI:"